portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase

```diff
@@ -567,16 +569,6 @@ definition {
 			ACProperties.switchProperty(propertyName = "${propertyName}");
```

Comment:
Hey @manuferreira, I think you need to use the `assignedPropertyName` variable:

`ACProperties.switchProperty(propertyName = "${assignedPropertyName}");`

Reply:
Oh yeah, I forgot to update this line.